tekton/ci-workspace/eventlistener.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: tekton-ci
+spec:
+  serviceAccountName: tekton-ci-listener
+  serviceType: NodePort
+  triggers:
+    - name: github-all-pull-request-ci
+      interceptors:
+        - github:
+            secretRef:
+              secretName: ci-webhook
+              secretKey: secret
+            eventTypes:
+              - pull_request
+        - cel:
+            filter: >-
+              body.repository.full_name.startsWith('afrittoli/') &&
+              body.repository.name in ['tektoncd-community', 'plumbing', 'pipeline', 'triggers', 'cli', 'dashboard', 'catalog', 'hub'] &&
+              (body.action == 'opened' ||
```

Comment:
I'm not very familiar with cell but is it possible to use the same construct for `body.action` as is used for `body.repository.name`?  In other words:

```
body.repository.full_name.startsWith('afrittoli/') &&
body.repository.name in ['tektoncd-community', 'plumbing', 'pipeline', 'triggers', 'cli', 'dashboard', 'catalog', 'hub'] &&
body.action in ['opened', 'synchronize', 'labeled', 'unlabeled', 'reopened']
```

The reason I suggest this is to just try and keep the syntax consistent and limited to 3 lines.

Reply:
Good point, thank you!